### ISSUE TITLE
Last.fm scrobbling fails and gets disabled (Your request timed out.)

### ISSUE THREAD

**Issue by dd23** [tiliado/nuvolaplayer#115]
I'm using the the developer version of Nuvola 3 with Google Music and have discovered the following problem with the last.fm plugin (I manually omitted the API keys):

```
[Runner:DEBUG    Nuvola] DeveloperSidebar.vala:120: Media player notify: artwork-file
[Runner:DEBUG    Nuvola] LastfmCompatibleScrobbler.vala:161: lastfm update now playing: Brother by Captain, We're Sinking
[Runner:DEBUG    Nuvola] LastfmCompatibleScrobbler.vala:246: Send request: api_key=AAAAA&artist=Captain%2C%20We%27re%20Sinking&method=track.updateNowPlaying&sk=SSSSS&track=Brother&api_sig=ZZZZZ&format=json
---------
<html>Your request timed out.   Please retry the request.                                                                                                                                                                                                                                                                                                                                                                                                                                                                  </html>
----------
[Runner:WARNING  Nuvola] AudioScrobblerComponent.vala:193: Update now playing failed for Last.fm (lastfm): <data>:1:1: Parse error: unexpected character `<', expected value
[Runner:WARNING  DioriteGtk] application.vala:91: Last.fm Error. Failed to update information about now playing track and this functionality has been disabled
[Runner:INFO     Nuvola] config.vala:260: Config saved to /home/dd/.config/nuvolaplayer3/apps_data/google_play_music/config.json
```

It appears, that Nuvola detects a problem with last.fm and disables the scrobbling completely.

What imho would be better is, that Nuvola detects the problem, stores the current scrobble to some stack/list and retries scrobbling after a certain time, say 30 or 60 seconds. This is also what the response from last.fm suggests.

Version info:
Google Music v5.3
```
Nuvola Player 3 Beta 3.0.0+201506062028.00f51f7
Revision 00f51f7bac10b2e79bdcf272a35771a30301d0ef
```

Edit: might be related to #74

**Comment by fenryxo** [comment]
Thanks for your bug report. Will probably implement similar fix like in #74.

**Comment by fenryxo** [comment]
I've implemented similar hack like in #74 Nuvola Player should now retry the scrobble request 20 times with 15 s delay. However, scrobbling is still disabled and the track is lost if it is enough. More robust failure handling might be implemented later #129.

Fixed in git master & 3.0.x branches. Will be included in the next update of package repository. Thanks for your bug report :+1:

---
Jiri Janousek, Nuvola Player Developer • My work on Nuvola Player and user support are [funded by Nuvola Patrons](https://tiliado.eu/nuvolaplayer/funding/). [Make a donation](https://tiliado.eu/nuvolaplayer/funding/) to keep the project going. Thank you! :heart:
